docs/pages/reference/networking.mdx

```diff
@@ -25,7 +25,7 @@ following use cases:
 - You want Teleport to issue an SSH certificate for the service with additional
   principals, e.g., host names.
 </TabItem>
-<TabItem scope={["cloud"]} label="Teleport Cloud">
+<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
```

Comment:
Won't this be the same for Team ?

Reply:
We have a lot of `TabItem`s like this in the docs, and I'm not really sure how to edit them in order to handle the introduction of Teleport Team. We've changed the "Teleport Cloud" usage to "Teleport Enterprise Cloud", which is what this change corrects, and I'm not sure we want to make "Teleport Cloud" a term for both Teleport Team and Teleport Enterprise Cloud. I suppose I can revert this change while we figure out what to do.

Reply:
I don't have a strong opinion on this, it just feels like if I was a "Teleport Team" subscriber and saw docs for "Teleport Enterprise Cloud" - I'd probably think they didn't apply to me.